csrc/attention/attention_kernels.cu

```diff
@@ -23,7 +23,11 @@
 
 #include <algorithm>
 
+#ifndef USE_ROCM
 #define WARP_SIZE 32
+#else
+#define WARP_SIZE 64
```

Comment:
To our knowledge the warp size of the MI200s (gfx9) supported in this port are 64, but may be different in other archs like in gfx10 where 32 is the default but 64 can also be used. In light of this we have updated the code so the warp size is decided by hip runtime which uses the default warp size unless configured otherwise.

Reply:
Thanks! Awesome!